app/assets/stylesheets/top-navbar.scss

```diff
@@ -11,6 +11,15 @@
     display: flex;
     height: $top-nav-bar-height;
   }
+
+  a {
+    padding: 3px;
+    border-bottom: 3px solid transparent;
```

Comment:
We've stopped doing border bottom on links now. We use `text-underline-position: under;` instead.

Reply:
Nice, thank you.